src/OptiHPLCHandler/empower_handler.py

```diff
@@ -173,6 +173,14 @@ def Status(self) -> List[HplcResult]:
         """Get the status of the HPLC."""
         raise NotImplementedError
 
+    def GetEmpowerProjects(self) -> list:
+        """
+        Assuming that the user has logged in in one project for example project = Mobile, this method fetches all available 
+        projectName that the user has access to and returns them as a list. 
+        """
+        project_list = self.connection.get("/authentication/project-list")[0]
+        return [item["projectName"] for item in project_list]
```

Comment:
Bruger vi nogensinde shortname til noget? Så kunne man lave et input det vælger om man vil have det ene eller det andet. Men lad os lade det ligge indtil vi ved at det skal bruges.

Reply:
jeg tror at i min web applikation bruger jeg short names til list. Det er nok en god ide at tilføje det. Jeg kommer til at opdatere yderligere for at tilpasse til behovet i min web applikation, så vi kommer nok til at opdatere det på et senere tidspunkt.